api/catalog/api/views/image_views.py

```diff
@@ -108,17 +107,10 @@ def oembed(self, request, *_, **__):
     def thumbnail(self, request, *_, **__):
         image = self.get_object()
 
-        image_url = image.url
+        image_url = image.thumbnail or image.url
```

Comment:
I am not sure we should use `thumbnail` for all providers just yet. Do we know what kind of thumbnail URLs we have in the catalog database? I'm afraid that because we sometimes used whatever the providers gave as `thumbnail`, the dimensions and the quality can vary greatly, and we might end up with low-quality thumbnails for some items.

Reply:
Sorry for the long delay in review, @krysal. Can we use `thumbnail` fallback only for the providers that we know have thumbnails of acceptable quality? @AetherUnbound, do you have any intuition/knowledge about the quality of thumbnails in the catalog database?